tests/sentry/utils/performance_issues/test_file_io_on_main_thread_detector.py

```diff
@@ -42,12 +46,41 @@ def create_proguard(self, uuid):
             f.writestr(f"proguard/{uuid}.txt", PROGUARD_SOURCE)
             create_files_from_dif_zip(f, project=self.project)
 
-    def test_gives_problem_correct_title(self):
-        event = EVENTS["file-io-on-main-thread"]
-
+    def find_file_io_on_main_thread_problems(
+        self, event: Event, setting_overides: Dict[str, Any] = None
+    ) -> List[PerformanceProblem]:
+        if setting_overides:
+            for option_name, value in setting_overides.items():
+                self.settings[DetectorType.FILE_IO_MAIN_THREAD][option_name] = value
```

Comment:
These settings overrides aren't used at any point, are they?

Reply:
They are used for the n plus one detector, I figured I would copy them over to be consistent and in case we expand tests.

Reply:
Maybe this should be part of some base class

Reply:
Yeah specific to the file it wasn't needed, your right it could be remove as there's no point. Gonna make one more cleanup PR.